### ISSUE TITLE
tqdm.keras: keras not found

### ISSUE THREAD

**Issue by arogers13** [tqdm/tqdm#885]
Hello! I am thrilled about the keras integration for tqdm. However, i've updated and I'm having trouble importing tqdm.keras. Has anyone else had issues with this?

```python
import tqdm, sys

print(tqdm.__version__, sys.version, sys.platform)
4.41.1 3.7.6 (default, Jan  8 2020, 20:23:39) [MSC v.1916 64 bit (AMD64)] win32

import tqdm.keras
Traceback (most recent call last):

  File "<ipython-input-18-4d4d30f13d41>", line 1, in <module>
    import tqdm.keras

  File "C:\Users\smnlab\Anaconda3\envs\tensorflow\lib\site-packages\tqdm\keras.py", line 4, in <module>
    from keras.callbacks import Callback

ModuleNotFoundError: No module named 'keras'

  ```

  [source website]: https://github.com/tqdm/tqdm/
  [known issues]: https://github.com/tqdm/tqdm/#faq-and-known-issues
  [issue tracker]: https://github.com/tqdm/tqdm/issues?q=
  [StackOverflow#tqdm]: https://stackoverflow.com/questions/tagged/tqdm

**Comment by casperdcl** [comment]
Looks like you haven't actually installed `keras`... you need to be able to `import keras` (rather than just `from tensorflow import keras`).

Should work if you `conda` or `pip install keras`.

Not sure if `tensorflow.keras` works; being discussed in https://github.com/tensorflow/addons/issues/919

**Comment by casperdcl** [comment]
right https://github.com/tqdm/tqdm/commit/6afc15966fc7a8b9a77fad2e23803b783e0debb3 should fix this (next release)

**Comment by shun-lin** [comment]
Hi,

If you are using tensorflow's version of keras in tf 2.0, you can use tensorflow addon's tqdm_progress_bar for now (https://github.com/tensorflow/addons/blob/master/docs/tutorials/tqdm_progress_bar.ipynb), iit you don't want to wait for the next release. There will be efforts merging tfa's progress bar w/ `tqdm.keras` soon. Hope this helps!

**Comment by casperdcl** [comment]
Released in `tqdm>=4.42.0`
